contrib/mesos/pkg/minion/server.go

```diff
@@ -333,6 +336,10 @@ func (ms *MinionServer) AddExecutorFlags(fs *pflag.FlagSet) {
 
 	// hack to forward log verbosity flag to the executor
 	fs.Int32Var(&ms.logVerbosity, "v", ms.logVerbosity, "log level for V logs")
+
+	// hacks to forward selected cadvisor flags to the executor
```

Comment:
... and in `launchExecutorServer` add some logic to add arguments `--global_housekeeping_interval=%v` and `--housekeeping_interval=%v` using the values from the minion struct.